src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -69,28 +68,37 @@ internal void InitializeLicense(string licenseText)
             license = foundLicense;
         }
 
-        static bool FirstTimeUser()
+        static bool NeedsReporting()
         {
             // Check for the presence of HKCU\Software\NServiceBus
-            using (var regNsbRoot = Registry.CurrentUser.OpenSubKey(@"Software\NServiceBus"))
+            using (var nsbRootRegKey = Registry.CurrentUser.OpenSubKey(@"Software\NServiceBus"))
             {
-                if (regNsbRoot == null)
+                if (nsbRootRegKey != null) return false;
+            }
+
+            //Check for the presence of HKCU\Software\ParticularSoftware
+            using (var particularRegKey = Registry.CurrentUser.OpenSubKey(@"Software\ParticularSoftware"))
+            {
+                // Check for the presence of HKCU\Software\ParticularSoftware\PlatformInstaller
+                using (var platformInstallerRegKey = particularRegKey?.OpenSubKey(@"Software\ParticularSoftware"))
```

Comment:
Copy paste/error? Based on the comment the subkey should `PlatformInstaller` not `Software\ParticularSoftware`